blog/2023-10-20-rebranding.mdx

```diff
@@ -0,0 +1,57 @@
+---
+slug: tolgee-rebranding
+title: "Tolgee's Exciting Transformation: A Modern, Fresh Look"
+description: Tolgee rebranding to new colors. We choose pink for it's modern and vibrant tone.
```

Comment:
meta description should be longer. How about: Experience Tolgee's exciting transformation – a vibrant new logo, new modern design of localization platform, and harmonious colors shaped by user feedback.